calculate_trigger.py

```diff
@@ -60,6 +101,16 @@ def search_for_fault_location(
             """Is current tb to short for trigger position"""
             if trigger_position > goldenrun_tb_info.at[idtbinfo, "ins_count"]:
                 idx = idx - 1
+                if idx < 0:
```

Comment:
This should be at line 58, since `idx` can also be decremented at line 88.

Reply:
Will do